external_deps/build.sh

```diff
@@ -745,6 +759,17 @@ setup_linux-amd64-default() {
 	common_setup
 }
 
+# Set up environment for arm Linux
+setup_linux-arm-default() {
```

Comment:
Is it ARM 32 or 64?

Reply:
32 bit, NaCl only supports 64-bit x86, 32-bit x86, 32-bit arm, 32-bit mips, no more.

Reply:
Shouldn't there be some indication of that in the arch string though since there can be more variants in the future?

Reply:
I expect 64bit arm to simply be named `arm64`, that's the way Debian names it. I had named 32-bit just `arm` because we have no reason to differentiates `arm` and `armhf`, as I assume arm-without-hardware-float is completely excluded from the list of platform we may add support for in the future…

Reply:
Actually I think calling it `armhf` would be worthwhile just so you can easily look up the name and understand what it is, even having no intention to port to `armel` or whatever.

Reply:
Done.